platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -45,15 +45,22 @@ OR
 
 ## What to expect from accessibility specialists during staging review
 
-After you request a staging review, the accessibility specialist reviews your work and documents any defects in GitHub/ZenHub tickets. Accessibility issues should be fixed according to the following schedule:
+After you request a staging review, the accessibility specialist will review your foundational tests. When the foundational tests have been verified, the specialist will review your product using several browser and screen reader combinations:
+
+- Windows 10 + IE11/Chrome + JAWS
+- Windows 10 + Firefox + NVDA
+- MacOS + Safari + VoiceOver
+- iOS + Safari + Mobile VoiceOver (as time permits)
+
+The accessibility specialist attaches the tickets to a single ZenHub epic and assigns the epic to the person who requested the staging review. During the staging review meeting, you and your team can ask questions about any of the issues identified. Applications are not considered Section 508 compliant until all severity 0, 1, 2, and 3 issues are remediated. Severity 4 items include minor issues that must be fixed, enhancements, and research.
```

Comment:
Are applications 508 compliant if they have severity 4 items? I was under the impression that they aren't but this reads like they are.

Reply:
Some of the 508 severity 4 items in the past were recommendations that didn't mean the app was out of compliance. I've added a `508-enhancement` tag to capture these things that don't rise to the level of defects.